src/JMS/Serializer/XmlSerializationVisitor.php

```diff
@@ -44,10 +44,14 @@ class XmlSerializationVisitor extends AbstractVisitor
     private $nullWasVisited;
     private $objectMetadataStack;
 
+    /** @var boolean */
+    private $formatOutput;
```

Comment:
set it default to false, please

we had some bugs fro such kind of implementation (https://github.com/schmittjoh/serializer/pull/317)